bluesky/run_engine.py

```diff
@@ -746,28 +734,26 @@ def _open_run(self, msg):
         self._clear_run_cache()
         self._run_start_uid = new_uid()
         self._run_start_uids.append(self._run_start_uid)
+        self.increment_scan_id()
```

Comment:
It always has. That is needlessly obfuscated, isn't it? It's only a four-line method. I will delete the method and put its code inline here.